docs/manuals/source/conf.py

```diff
@@ -270,12 +271,13 @@
     re.compile(r"^master$"),
     re.compile(r"^bareos-18.2$"),
     re.compile(r"^bareos-19.2$"),
+    re.compile(r"^bareos-20.2$"),
 )
 scv_whitelist_tags = (re.compile(r"^not-exisiting-tag$"),)
 scv_show_banner = True
 scv_priority = "branches"
-scv_root_ref = "bareos-19.2"
-scv_banner_main_ref = "bareos-19.2"
+scv_root_ref = "bareos-20.2"
+scv_banner_main_ref = "bareos-20.2"
```

Comment:
we must not merge that change before release of bareos 20 as it will change the default documentation version